libsolidity/formal/BMC.cpp

```diff
@@ -605,6 +636,14 @@ void BMC::checkUnderflow(BMCVerificationTarget& _target, smtutil::Expression con
 			_target.type == VerificationTarget::Type::UnderOverflow,
 		""
 	);
+
+	if (
+		m_solvedTargets.count(_target.expression) &&
+		(m_solvedTargets.at(_target.expression).count(VerificationTarget::Type::Underflow) ||
+			m_solvedTargets.at(_target.expression).count(VerificationTarget::Type::UnderOverflow))
+	)
```

Comment:
Strictly speaking that's coding style violations and it'd need to be something like:
```suggestion
	if (
		m_solvedTargets.count(_target.expression) && (
			m_solvedTargets.at(_target.expression).count(VerificationTarget::Type::Underflow) ||
			m_solvedTargets.at(_target.expression).count(VerificationTarget::Type::UnderOverflow)
		)
	)
```
Or something like that...
We should really have a helper for this, we do this weird stuff all the time - ideally this would be

```cpp
if (
	auto const* target = getMemberIfExists(m_solvedTargets, _target.expression);
	target && (
		contains(*target, VerificationTarget::Type::Underflow) ||
		contains(*target, VerificationTarget::Type::Underflow)
	)
)
```

Reply:
Done without the helper